interviews.md

```diff
@@ -64,16 +64,16 @@ title: "Archive of King Gizzard interviews"
 
 ### 2017
 
-* Dec 6: Consequence, [Band of the Year King Gizzard and the Lizard Wizard Can’t Stop, Won’t Stop, and Don’t Need to Stop](https://consequence.net/2017/12/band-of-the-year-king-gizzard-and-the-lizard-wizard-cant-stop-wont-stop-and-dont-need-to-stop/)
-* Nov 27: (print) Beat, "Welcome to Gizzfest" [page 1](https://preview.redd.it/rox0bbpbkbda1.jpg?width=3024&format=pjpg&auto=webp&v=enabled&s=535e49e11002514606802a6e71399c0d0f942091), [page 2](https://preview.redd.it/x4vwuapbkbda1.jpg?width=3024&format=pjpg&auto=webp&v=enabled&s=019eb7f8a76341fd4c7a50681b2bf4e1fd98ffab), [page 3](https://preview.redd.it/brqebbpbkbda1.jpg?width=3024&format=pjpg&auto=webp&v=enabled&s=a3505ddd0670e05db4a78b57c0fab485e6c8c74b), by Tom Parker
+* Dec 6: Consequence, [Band of the Year King Gizzard and the Lizard Wizard Can’t Stop, Won’t Stop, and Don’t Need to Stop](https://consequence.net/2017/12/band-of-the-year-king-gizzard-and-the-lizard-wizard-cant-stop-wont-stop-and-dont-need-to-stop/) - [(archived)](https://web.archive.org/web/20230325211958/https://consequence.net/2017/12/band-of-the-year-king-gizzard-and-the-lizard-wizard-cant-stop-wont-stop-and-dont-need-to-stop/)
+* Nov 27: (print) Beat, "Welcome to Gizzfest" [page 1](https://preview.redd.it/rox0bbpbkbda1.jpg?width=3024&format=pjpg&auto=webp&v=enabled&s=535e49e11002514606802a6e71399c0d0f942091) - [(archived)](https://web.archive.org/web/20230121035500/https://preview.redd.it/rox0bbpbkbda1.jpg?width=3024&format=pjpg&auto=webp&v=enabled&s=535e49e11002514606802a6e71399c0d0f942091), [page 2](https://preview.redd.it/x4vwuapbkbda1.jpg?width=3024&format=pjpg&auto=webp&v=enabled&s=019eb7f8a76341fd4c7a50681b2bf4e1fd98ffab) - [(archived)](https://web.archive.org/web/20230325211655/https://preview.redd.it/x4vwuapbkbda1.jpg?width=3024&format=pjpg&auto=webp&v=enabled&s=019eb7f8a76341fd4c7a50681b2bf4e1fd98ffab), [page 3](https://preview.redd.it/brqebbpbkbda1.jpg?width=3024&format=pjpg&auto=webp&v=enabled&s=a3505ddd0670e05db4a78b57c0fab485e6c8c74b) - [(archived)](https://web.archive.org/web/20230121035502/https://preview.redd.it/brqebbpbkbda1.jpg?width=3024&format=pjpg&auto=webp&v=enabled&s=a3505ddd0670e05db4a78b57c0fab485e6c8c74b), by Tom Parker
```

Comment:
Might be able to use Archive to stitch these three photos together?? (...but maybe no, the photographer would have to give explicit permission?)

Reply:
That sounds cool. (Although I have no idea how to do it.)